Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [workspace]
 members = [
-    "runner",
-    "runner-macros",
+    "workspaces",
+    "workspaces-macros",
```

Comment:
Might want to squat these on crates.io, they don't currently exist but good to be sure

Reply:
good call. Should we go with purely `workspaces` or `near-workspaces`?

Reply:
idk I like the simplicity of `workspaces` but probably best to get others opinions if it would be better to have explicit near naming

cc @DiscRiskandBisque @jberrytech